### PR TITLE
try pining setuptools

### PR DESCRIPTION
**Pull request by auvipy**

### CODE REVIEW

File: requirements/pkgutils.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-setuptools>=20.6.7
+setuptools==58.2.0
```
Comment (pawl) [inline requirements/pkgutils.txt:1]: Pinning `setuptools<59.8.0` worked for me.

Comment (auvipy) [reply requirements/pkgutils.txt:1]: i think i have to restrict the version in github action as well

Comment (auvipy) [reply requirements/pkgutils.txt:1]: see https://github.com/celery/kombu/blob/master/.github/workflows/ci.yaml#L21

Comment (auvipy) [reply requirements/pkgutils.txt:1]: https://github.com/celery/kombu/pull/1472

**Comment by auvipy** [comment]
confused CI

**Pull request merged**